/**:
  ros__parameters:
    position_tolerance: 0.1
    sit_duration: 5.0
    x_coords: [0.98, 0.93, 0.00, 2.00, 0.00, 2.00, 1.00, 0.00, 2.00,  1.86, -0.21]
    y_coords: [0.16, 5.30, 5.52, 5.52, 15.0, 15.0, 15.0, 15.8, 15.8, -1.05, -1.05]